locales/uk/translation.json

```diff
@@ -259,8 +259,8 @@
       "hide-in-tab-search-form": "Esc",
       "jump-to-next-search-occurance": "Enter",
       "jump-to-previous-occurance": "Shift + Enter",
-      "previous-chapter": "Left",
-      "next-chapter": "Right",
+      "previous-chapter": "←",
+      "next-chapter": "→",
```

Comment:
@tobias-klein FYI: I think this Unicode character is the simplest way to refer to those keys in Ukrainian and Russian, otherwise you have to use 2-3 words: "Cursor to the left", "Cursor to the right" which is confusing or two word "Left Key" which might be okay...